src/Symfony/Component/ExpressionLanguage/ParserCache/ArrayParserCache.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * @author Adrien Brault <adrien.brault@gmail.com>
+ *
+ * @deprecated ParserCacheInterface is deprecated. Use Symfony\Component\Cache\Adapter\ArrayAdapter instead
```

Comment:
missing deprecation triggering just below the namespace declaration (see other cases in the code base).
Note that we don't add these on interfaces to nothing to add on ParserCacheInterface.php